docs/program/apis/cloud.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Managing Your Fleet with Viam's Cloud API"
+linkTitle: "Cloud Management"
+weight: 20
+type: "docs"
+description: "How to use the cloud API with Viam's client SDKs."
+tags: ["cloud", "sdk", "viam-server", "networking", "apis", "robot api", "cloud management"]
+---
+
+The cloud app API allows you to [manage your robot fleet](/manage/fleet/) with code instead of through the [Viam app](https://app.viam.com/).
+Functionality includes:
+
+- create and manage organizations, locations, and individual robots
+- manage permissions and authorization
+- create and manage fragments
+
+{{% alert title="Support Notice" color="note" %}}
+
+As of August 24, 2023, cloud app API methods are only available in the Python SDK.
```

Comment:
```suggestion
Cloud app API methods are only available in the Python SDK.
```
I think we have mostly been having these notices be like this in the past few months so that we don't have a glaring "THIS PAGE IS OUT OF DATE AND WE DON'T UPDATE OUR DOCS" if it gets outta sync 🤣 I'm fine w it either way though